swan-lake/why-ballerina/graphical.md

```diff
@@ -1,67 +1,37 @@
+In today’s cloud-era, you need technologies that can model distributed systems in a more developer-friendly way. The feature-rich development experience of the [Ballerina Visual Studio code extension](/vscode/) brings in graphical representations for designing the architecture of your applications, designing and editing your services, modeling how concurrent execution flows and how multiple actors interact with each other, and mapping the data conversions within your applications.
 
-That’s why sequence diagrams are the foundation for designing the syntax and semantics of the Ballerina language. Ballerina provides the flexibility of a general-purpose language while having features to model and visualize solutions based on higher-level abstractions derived from sequence diagrams.
+Ballerina provides the flexibility of a general-purpose language with the capabilities to model and visualize solutions based on higher-level abstractions derived from the graphical representations provided by its developer tooling features as described below.
 
+## Sequence Diagram View
 
-## Sequence diagrams in Ballerina
+The Sequence Diagram View provided by the Ballerina VS Code extension allows you to visualize a program written in Ballerina as a sequence diagram. The diagram will display the logic and network interaction of a function or a service resource. You can view and edit these diagrams using the Ballerina VS Code extension.
 
-Being based on sequence diagrams, Ballerina allows you to visualize a program written in Ballerina as a sequence diagram. The diagram will display the logic and network interaction of a function or a service resource. You can view and edit these diagrams using the Ballerina VSCode plugin.
+For more information, see [Sequence Diagram View](https://wso2.com/ballerina/vscode/docs/implement-the-code/sequence-diagram-view/).
 
-<img src="/img/why-pages/sequence-diagrams-for-programming-1.png" alt="Ballerina sequence diagram"  style="width:100%; border: 1px solid #eee;">
+## Architecture View
 
-One of the key benefits of the diagram is that it acts as a documentation of the code. It makes it far easier to comprehend the program than reading the source code. Even if you are not familiar with Ballerina code syntax it is easier to understand the diagram.
+The Architecture View of the Ballerina VS Code extension provides a holistic picture of the components and their interactions in Ballerina projects purely based on the source code. It provides several types of diagrams namely, the Service Diagram Level 1, Service Diagram Level 2, type diagram, and composition diagram to explore and ease the designing of the application.  
 
-> ***"[With Ballerina] you can get sequence diagrams automatically. When things start to get complicated and you need to understand and socialize with the rest of your team what it is that you're building, these diagrams become very helpful." -- Christian Posta, field CTO, solo.io***
```

Comment:
Why removing this.

Reply:
Removed to maintain consistency with the other sections in which it should be sufficient to provide a single-paragraph intro about the topic and this more information regarding the Sequence Diagram View should be there in the [VS Code docs](https://wso2.com/ballerina/vscode/docs/implement-the-code/sequence-diagram-view/).